sdk-core/src/main/java/care/data4life/sdk/migration/CompatibilityEncoder.kt

```diff
@@ -36,17 +36,16 @@ internal object CompatibilityEncoder : MigrationInternalContract.CompatibilityEn
         return result
     }
 
-    override fun encode(tagValue: String): Triple<String, String, String> {
-        val validEncoding = tagEncoding.encode(tagValue)
+    override fun encode(tagValue: String): MigrationInternalContract.CompatibilityTag {
         val kmpLegacyEncoding = tagEncoding.normalize(tagValue)
-        val jsLegacyEncoding = mapJSExceptions(
-            urlEncoding.encode(kmpLegacyEncoding)
-        )
 
-        return Triple(
-            validEncoding,
-            kmpLegacyEncoding, // aka KMPLegacyTags
-            jsLegacyEncoding
+        return MigrationInternalContract.CompatibilityTag(
```

Comment:
happy you moved on from the triple, wondering about the naming but i dont have a big opinion there. Maybe it would help to specify that is just for search but i wold be ok also to keep it this way.

Reply:
done